src/Cypher.ts

```diff
@@ -45,7 +45,8 @@ export { Path, NamedPath } from "./references/Path";
 export { PropertyRef as Property } from "./references/PropertyRef";
 
 // Expressions
-export { Exists } from "./expressions/Exists";
+export { Exists } from "./expressions/subquery/Exists";
+export { Count } from "./expressions/subquery/Count";
```

Comment:
It's also exported `count` (the aggregation function), are we happy with that?

Reply:
Yeah, that is tricky indeed.

Luckily our model of clauses -> classes and functions -> functions makes a clear distinction of these two. But it is still confusing. Not sure we can do anything about it, as these are also confusing in Cypher itself :sweat_smile:

Reply:
Not really, alternative we could go with:
- Using different namespaces.
- Change the export name.

But I would have gone with this solution as well, until works!